packages/amplify_authenticator/lib/src/utils/validators.dart

```diff
@@ -117,6 +117,9 @@ String? validatePhoneNumber(String? phoneNumber) {
   if (phoneNumber == null || phoneNumber.isEmpty) {
     return 'Phone number cannot be empty';
   }
+  if (!phoneNumberRegex.hasMatch(phoneNumber)) {
+    return 'Invalid phone number format';
```

Comment:
We need to go through these validators and add localization. I can take that in a separate PR.